src/main/webapp/assets/modules/talent-search/search-result/search-result.con.js

```diff
@@ -1,6 +1,7 @@
 techlooper.controller("tsSearchResultController",
   function ($scope, $timeout, tsMainService, tsSearchResultService, $routeParams, $http, jsonValue, $location) {
     var request = {};
+    var loopresPoint = 'Loopers Rank is a measure of profile completeness, relevance and experience; the more points means profile contains more personal information, person skills’ are more relevant to market demand and this person has contributed more to open source projects';
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>'loopresPoint' is defined but never used.